ProteGO/Common/EncountersManager/EncountersManager.swift

```diff
@@ -2,6 +2,9 @@ import Foundation
 import RealmSwift
 
 final class EncountersManager: EncountersManagerType {
+    // NOTE: Move somewhere else...
+    var lastExpiringBeaconId: ExpiringBeaconId?
```

Comment:
Sugestia: Oprócz przeniesienia tej property gdzie indziej, może warto rozważyć zmianę nazwy (zarówno pola jak i struktury ExpiringBeaconId)? O ile dobrze rozumiem flow, to `ExpiringBeaconId` jest używany tylko jako reprezentacja naszego urządzenia i jest używany jedynie do rozgłaszania przed Advertiser. Może lepszą nazwą byłoby np. `DeviceBeaconId` czy `LocalBeaconId`? Może jeszcze lepiej byłoby porzucić `Id` i mieć `LocalBeacon` tak by nazwa błędnie nie sugerowała hierarchii dziedziczenia pomiędzy `ExpiringBeaconId`/`DeviceBeaconid`/`LocalBeaconId` a `BeaconId`.

Reply:
`Expiring` miało sugerować powiązanie daty z `BeaconId`, który ma swoją ważność oraz dać mechanizm, który gwarantuje sprawdzenie daty ważności przy wyciąganiu `BeaconId`. Jak zawsze, jestem otwarty na zmiany w nazewnictwie.

Reply:
Głównie chodzi mi o to, że `Expiring` nie sugeruje, że jest to id naszego urządzenia (popraw mnie jeśli się mylę co do tego). W sytuacji, gdzie mamy komunikację w dwie strony - wysyłamy i odbieramy id innych urządzeń, myślę że warto mieć jasne rozgraniczenie.

Reply:
Tak, aktualnie `BeaconId` może zawierać lokalne i zdalne ID. Można wprowadzić dodatkowe typy sugierujące, czy są one lokalne/zdalne. Dobrze by to było zrobić bez duplikacji kodu. Jakieś sugestie?

Reply:
Może enum `LocalBeaconId` i `RemoteBeaconId` z których oba by zawierały `BeaconId`, jeden `expirationDate` a drugi `encounterDate`?

Reply:
Advertiser (Local Beacon) i Scanner (Remote Beacons) nie są ze sobą w żaden sposób powiązane, więc te id też raczej nie powinny być ze sobą powiązane w postaci jednego Enuma. 

W tym momencie mamy ProteGoDevice które reprezentuje inne urządzenia, więc może wystarczy mieć `LocalDevice` które będzie naszym devicem? Wtedy nazewnictwo itp jest spójne

Reply:
Może warto otworzyć na to issue i nie przeciągać (i tak zbyt dużego) PR?

Reply:
Dodałem https://github.com/ProteGO-app/ios/issues/49 @SebastianOsinski